test/hotspot/jtreg/applications/jcstress/accessAtomic.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
It is not needed to update copyrights if nothing was changed in this file. Please just remove such files from commit.

Reply:
Got it!
All the wrappers have been reverted, the only change is the JcstressRunner, depending on the new jcstress version.